Flipper/Packages/UI/Sources/Hub/Apps/Components/AppsList.swift

```diff
@@ -36,6 +41,9 @@ struct AppList: View {
                         application: application,
                         isInstalled: isInstalled
                     )
+                    .onAppear {
+                        isLoading = (application == applications.last)
```

Comment:
we already check the state in its parent, so we can just call loadMore if application == applications.last